src/frontend/src/flows/displayUserNumber.ts

```diff
@@ -1,12 +1,24 @@
 import { html, render } from "lit-html";
+import { warningIcon } from "../components/icons";
 
 const pageContent = (userNumber: bigint) => html`
   <div class="container">
     <h1>Congratulations!</h1>
     <p>
-      Please record your new Identity Anchor. You will need it later to use
-      Internet Identity or to add additional devices.
+     Your new Identity Anchor has been created. You will need it later to use
+     Internet Identity or to add additional devices.
     </p>
+    <div class="nagBox">
+      <div class="nagIcon">${warningIcon}</div>
+      <div class="nagContent">
+        <div class="nagTitle">Record Your Identity Anchor</div>
+        <div class="nagMessage">
+          Do <b>NOT</b> forget to save this Identity Anchor. Save a backup on a
+          storage medium and write it down.</br>
+          Note: It is advised to keep Identity Anchors secret in order to stay anonymous.
```

Comment:
What do you mean by "stay anonymous"? Anchors are never shared when authenticating

Reply:
Ah, right.